modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts

```diff
@@ -280,7 +280,7 @@ export type Challenge = {
   h2: string;
 };
 
-export type backupGpgKey = SerializedKeyPair<string> | Key;
```

Comment:
Nice catch!!!, But I think this can be considered as a breaking change since its exported out. Can you add the breaking change into the commit ([Message format](https://www.conventionalcommits.org/en/v1.0.0/#:~:text=in%20Semantic%20Versioning\).-,BREAKING,-CHANGE%3A%20a))